config/locales/spotlight.en.yml

```diff
@@ -1,4 +1,9 @@
 en:
+  time:
+    formats:
+      date:
+        formats:
+          default: '%Y-%m-%d %l:%M%P'
```

Comment:
if @ggeisler is ok with the `:short` format, i'll pull this section and switch over to that and adjust tests accordingly.  otherwise, i can just add a new key and call that.  i might be slightly in favor of the latter, but i don't feel strongly about it at all.